grapi/backend/kopano/message.py

```diff
@@ -56,6 +56,16 @@ def update_attr_value(item, attr_name, value):
     setattr(item, attr_name, value)
 
 
+def set_categories(item, value):
```

Comment:
Please use `update_attr_value` to set or update attributes. So, this function is no longer needed.

Reply:
Should be fixed in https://github.com/Kopano-dev/grapi/pull/4/commits/ad62849ee1aa29192de26dc0751fa7959b05cccf